Crashes on launch

I just updated the app to latest `1.72.0 (166)` version from F-Droid now it crashes instantly upon launch.

<details><summary>Logs:</summary>
<p>


```
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: FATAL EXCEPTION: main
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: Process: de.markusfisch.android.binaryeye, PID: 5577
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: java.lang.RuntimeException: Unable to resume activity {de.markusfisch.android.binaryeye/de.markusfisch.android.binaryeye.activity.CameraActivity}: java.lang.IllegalArgumentException: No enum constant de.markusfisch.android.zxingcpp.ZxingCpp.BarcodeFormat.DATA_BAR
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3805)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3845)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3065)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.ActivityThread.-wrap11(Unknown Source:0)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1724)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7000)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:441)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: Caused by: java.lang.IllegalArgumentException: No enum constant de.markusfisch.android.zxingcpp.ZxingCpp.BarcodeFormat.DATA_BAR
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at java.lang.Enum.valueOf(Enum.java:257)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at de.markusfisch.android.zxingcpp.ZxingCpp$BarcodeFormat.valueOf(r8-map-id-2e45c4f463f015b6435fd4e6afee406b66e656582530b887dba676b92db3f920:3)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at wi.c(r8-map-id-2e45c4f463f015b6435fd4e6afee406b66e656582530b887dba676b92db3f920:35)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at de.markusfisch.android.binaryeye.activity.CameraActivity.A(r8-map-id-2e45c4f463f015b6435fd4e6afee406b66e656582530b887dba676b92db3f920:17)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at de.markusfisch.android.binaryeye.activity.CameraActivity.v(r8-map-id-2e45c4f463f015b6435fd4e6afee406b66e656582530b887dba676b92db3f920:18)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at de.markusfisch.android.binaryeye.activity.CameraActivity.onResume(r8-map-id-2e45c4f463f015b6435fd4e6afee406b66e656582530b887dba676b92db3f920:72)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1363)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.Activity.performResume(Activity.java:7485)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3780)
11.04 04:08:15.347 10190 5577 5577 de.markusfisch.android.binaryeye AndroidRuntime: 	... 10 more
```

</p>
</details>

Device details:
Android version - 8.1.0
ROM vendor - Samsung stock
Device model - SM-G610F (Samsung J7 Prime)

--- a/app/src/main/kotlin/de/markusfisch/android/binaryeye/zxingcpp/Mapping.kt
+++ b/app/src/main/kotlin/de/markusfisch/android/binaryeye/zxingcpp/Mapping.kt
@@ -10,6 +10,7 @@ val oldToNewFormatNames = mapOf(
 	"PZN" to "PZN",
 	"CODE_93" to "Code93",
 	"CODE_128" to "Code128",
+	"DATA_BAR" to "DataBar",
 	"DATA_BAR_OMNI" to "DataBarOmni",
 	"DATA_BAR_STK" to "DataBarStk",
 	"DATA_BAR_LTD" to "DataBarLtd",

